lib/rake/test.rake

```diff
@@ -11,6 +11,11 @@ namespace :test do
     TestRunUtils.run_apps_tests
   end
 
+  desc 'Run a single eyes test.'
```

Comment:
Worth calling out 'local' or 'chromedriver' in this description, and in the documentation above?  I guess 'local' has two meanings for UI tests (testing dashboard running on localhost, or using a local browser to test it) but this is local in both senses.